qlib/rl/integration/feature.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""
```

Comment:
Suggest moving this file into `order_execution/integration.py`.

Okay to keep it for now.